polytiming.js

```diff
@@ -1,17 +1,28 @@
 (function() {
   'use strict';
 
-  let measuredElements = new Set();
-  let measuredMethods = new Set();
-  let configuredMethods = window.location.search.slice(1).split('&')
-      .map(function(part) {
-        return part.split('=');
-      }).reduce(function(l, r) {
-        if (r[0] === 'instrumentPolymer') {
-          return l.concat(r[1].split(','));
-        }
-        return l;
-      }, []);
+  function setOfQueryParams(paramName) {
+    return new Set(window.location.search.slice(1).split('&')
+        .map(function(part) {
+          return part.split('=');
+        }).reduce(function(l, r) {
+          if (r[0] === paramName) {
+            return l.concat(r[1].split(','));
+          }
+          return l;
+        }, []));
+  }
+
+  const measuredElements = new Set();
+  const measuredMethods = new Set();
+  const configuredMethods = setOfQueryParams('instrumentPolymer')
+  const trackedElements = setOfQueryParams('trackElement')
+  let shouldTrackElement = (elementName) => true;
```

Comment:
In Chrome at least, `Set` objects are 3x faster on my machine: https://albertxing.com/stool/#c028c3dca7f423095c19597cc206f9c7

(Assuming you're talking about converting trackedElements and not the shouldTrackElement function)